app/validators/url_validator.rb

```diff
@@ -1,10 +1,14 @@
 class UrlValidator < ActiveModel::EachValidator
-    def validate_each(record, attribute, value)
-      record.errors[attribute] << (options[:message] || "must be a valid URL") unless url_valid?(value)    
+  def validate_each(record, attribute, value)
+    record.errors[attribute] << (options[:message] || 'must be a valid URL') unless url_valid?(value)
```

Comment:
It's considered a good practice to move static strings to a constants. Also - don't forget to freeze it(`freeze` or `#frozen_string_literal: true` at the top of the file) to prevent unnecessary memory allocation